### PR TITLE
fix: Change buttons and tables

### PR DESCRIPTION
**Pull request by mmaillart**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/Talend/bootstrap-theme/blob/master/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
 
The Button and List components do not respect the UI Specs (default color, hover and size). 

**What is the new behavior?**

The Button and List components respect the UI Specs (default color and size)

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/theme/_tables.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+// Hover effect
+//
+// Placed here since it has to come after the potential zebra striping
+
+.table-hover {
+  > tbody > tr:hover {
+    background-color: $table-bg-hover;
+    font-weight: 700;
+    cursor: pointer;
```
Comment (jmfrancois) [inline src/theme/_tables.scss]: should be in the List component I guess. not here

File: src/theme/_variables.scss [hunk 2]
```diff
@@ -164,13 +164,17 @@ $table-border-color:            #DDD !default;
 $btn-font-weight:                normal !default;
 
 $btn-default-color:              #333 !default;
-$btn-default-bg:                 #FFF !default;
+$btn-default-bg:                 #dedede !default;
```
Comment (jmfrancois) [inline src/theme/_variables.scss]: capital please

File: src/theme/_tables.scss (hunk 1 above)
Comment (jmfrancois) [inline src/theme/_tables.scss]: should be in the List component I guess. not here

File: src/theme/_navbar.scss [hunk 3]
```diff
@@ -25,4 +25,8 @@ $navbar-brand-logo-width: 90px !default;
 			}
 		}
 	}
+  .navbar-form {
+	margin-top: 4px;
```
Comment (mmaillart) [inline src/theme/_navbar.scss]: I changed the default height of the buttons so comply qith our guidelines. Therefore, the Search bar was not aligned anymore in the App Header

File: src/theme/_buttons.scss [hunk 4]
```diff
@@ -81,3 +81,19 @@ $btn-background-tint-percent: 10 !default;
 		}
 	}
 }
+
+
+// Alternate buttons
+// --------------------------------------------------
+
+.btn-primary {
+  @include button-variant($btn-secondary-color, $btn-secondary-bg, $btn-secondary-border);
```
Comment (mmaillart) [inline src/theme/_buttons.scss]: OK, I changed the Secondary color by a Validation color

File: src/theme/_variables.scss (hunk 2 above)
Comment (mmaillart) [inline src/theme/_variables.scss]: Done

File: src/theme/_tables.scss (hunk 1 above)
Comment (mmaillart) [inline src/theme/_tables.scss]: Ok

File: src/theme/_tables.scss [hunk 5]
```diff
@@ -0,0 +1,30 @@
+////
+/// Bootstrap navbars customization
+/// @group Navs
+////
+
+.table {
```
Comment (mmaillart) [inline src/theme/_tables.scss]: The PR is also about Table, my bad...

Comment (frassinier) [inline src/theme/_tables.scss]: (In fact, each pull request must should concern one feature)

**Pull request merged**